kms.tf

```diff
@@ -1,3 +1,9 @@
+locals {
+  provisioner_role_arn         = var.assume_role_arn == null ? data.aws_caller_identity.current.arn : var.assume_role_arn
+  cloudtrail_key_policy_root   = var.is_existing_cloudtrail_cross_account == false ? "arn:aws:iam::${local.customer_aws_account_id}:root" : "arn:aws:iam::${var.existing_cloudtrail_log_bucket_account_id}:root"
```

Comment:
shouldn't this always be the cloudtrail account & not the log bucket account? probably the `else` part is never applied since we don't even create the cloudtrial key for existing cloudtrails?

Reply:
good point, I'll correct as a follow-on task but would rather not introduce new variables rn. I'll create a ticket to validate / remove.